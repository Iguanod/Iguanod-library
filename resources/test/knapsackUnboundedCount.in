333
0.819 0.314 1.115 1.023 1.58 2.98 3.046 4.285 5.618 6.042 7.605 9.05 11.038 12.555 14.069 16.17 17.804 19.933 22.272 24.196 27.372 29.758 31.729 35.343 37.814 40.914 44.15 47.286 50.647 54.478 58.501 62.004 65.726 69.859 73.455 78.115 82.903 87.347 91.422 96.402 100.887 106.319 111.588 116.52 121.637 127.086 132.33 138.201 143.893 150.536 156.134 162.608 168.95 175.507 181.682 187.909 194.583 202.053 209.078 216.507 223.114 230.627 238.284 245.863 253.955 261.49 269.71 277.276 285.631 293.561 302.554 310.534 319.174 328.3 336.946 346.857 355.033 365.155 374.141 384.152 393.062 403.573 413.034 423.184 432.989 443.545 454.025 463.902 474.753 485.208 496.867 506.842 518.476 530.071 541.413 552.199 564.404 576.002 587.377 598.902 611.077 623.264 636.061 647.796 661.141 673.505 685.739 698.797 711.671 724.819 737.844 751.728 764.627 778.67 792.337 806.395 820.052 834.76 848.693 862.34 876.939 891.644 906.748 921.618 936.278 951.247 966.226 981.915 996.634 1012.251 1028.181 1044.259 1059.764 1075.688 1091.688 1107.644 1124.198 1140.985 1157.122 1173.861 1191.42 1208.13 1225.034 1241.955 1259.255 1276.587 1294.141 1311.8 1330.359 1348.1 1366.268 1384.434 1402.223 1420.513 1439.084 1457.936 1476.971 1495.343 1514.368 1533.867 1552.307 1571.864 1591.14 1611.472 1630.381 1650.148 1670.41 1690.141 1711.003 1730.594 1751.482 1771.898 1792.598 1813.207 1834.725 1855.008 1876.52 1898.065 1919.468 1940.148 1961.874 1984.189 2006.205 2027.977 2049.772 2072.259 2094.627 2117.342 2139.058 2162.485 2185.077 2207.82 2230.859 2254.218 2277.893 2300.755 2323.998 2347.8 2371.783 2395.471 2420.199 2443.529 2467.813 2492.885 2517.246 2541.545 2566.768 2591.293 2615.708 2640.919 2666.715 2691.742 2717.639 2743.211 2768.312 2793.989 2819.705 2846.274 2872.028 2898.754 2925.499 2951.178 2978.73 3004.652 3031.44 3058.653 3086.266 3112.903 3140.529 3168.637 3195.55 3223.712 3251.41 3279.107 3307.36 3335.973 3363.653 3392.579 3420.511 3449.772 3478.872 3507.383 3536.345 3565.049 3594.366 3624.032 3653.478 3683.606 3713.211 3742.81 3772.594 3803.634 3833.354 3863.991 3894.626 3924.807 3955.085 3985.945 4017.097 4048.021 4079.441 4110.742 4142.489 4173.846 4205.485 4237.8 4269.496 4301.787 4333.127 4365.5 4397.717 4430.547 4463.139 4495.705 4528.837 4561.56 4595.27 4628.577 4661.647 4695.225 4728.921 4762.129 4796.296 4830.408 4864.688 4898.5 4933.032 4966.998 5002.152 5036.025 5071.307 5106.513 5140.721 5176.788 5211.517 5247.029 5281.987 5318.516 5353.856 5389.975 5426.025 5462.314 5497.727 5534.684 5571.267 5606.992 5643.883 5680.901 5718.158 5755.456 5792.05 5829.89 5866.519 5904.918 5942.033 5980.141 6017.717 6055.473 6094.47 6132.166 6170.998 6208.949 6248.218 6286.318 6325.079 6364.791 6403.045 6443.106 6481.942 6521.133 6560.727 6601.002 6641.022 
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 23 26 28 31 34 37 40 43 46 50 53 57 61 65 69 73 77 81 86 91 95 100 105 110 115 121 126 132 137 143 149 155 161 168 174 181 187 194 201 208 215 222 230 237 245 252 260 268 276 285 293 301 310 319 327 336 345 355 364 373 383 392 402 412 422 432 442 453 463 474 485 495 506 517 529 540 551 563 575 586 598 610 622 635 647 660 672 685 698 711 724 737 751 764 778 791 805 819 833 847 862 876 891 905 920 935 950 965 981 996 1011 1027 1043 1059 1075 1091 1107 1123 1140 1156 1173 1190 1207 1224 1241 1258 1276 1293 1311 1329 1347 1365 1383 1401 1420 1438 1457 1475 1494 1513 1532 1552 1571 1590 1610 1630 1650 1670 1690 1710 1730 1750 1771 1792 1812 1833 1854 1875 1897 1918 1940 1961 1983 2005 2027 2049 2071 2093 2116 2138 2161 2184 2207 2230 2253 2276 2300 2323 2347 2371 2395 2419 2443 2467 2491 2516 2541 2565 2590 2615 2640 2665 2691 2716 2742 2767 2793 2819 2845 2871 2898 2924 2951 2977 3004 3031 3058 3085 3112 3140 3167 3195 3222 3250 3278 3306 3335 3363 3391 3420 3449 3477 3506 3535 3565 3594 3623 3653 3682 3712 3742 3772 3802 3832 3863 3893 3924 3955 3985 4016 4047 4079 4110 4141 4173 4205 4236 4268 4300 4332 4365 4397 4430 4462 4495 4528 4561 4594 4627 4661 4694 4728 4761 4795 4829 4863 4897 4932 4966 5001 5035 5070 5105 5140 5175 5211 5246 5281 5317 5353 5389 5425 5461 5497 5533 5570 5606 5643 5680 5717 5754 5791 5828 5866 5903 5941 5979 6017 6055 6093 6131 6170 6208 6247 6285 6324 6363 6402 6442 6481 6520 6560 6600 6640 
12853
