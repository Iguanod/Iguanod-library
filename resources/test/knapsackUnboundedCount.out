15063.358
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 675 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 
